pkg/datapath/loader/loader_test.go

```diff
@@ -51,7 +51,7 @@ var (
 	dirInfo *directoryInfo
 	ep      = testutils.NewTestEndpoint()
 	hostEp  = testutils.NewTestHostEndpoint()
-	bpfDir  = filepath.Join(testutils.CiliumRootDir, "bpf")
+	bpfDir  = filepath.Join("..", "..", "..", "bpf")
```

Comment:
Will this work regardless of where `go test` is invoked from?

That is, will it work both when running `go test ./...`  from the top dir,  `go test` inside `pkg/datapath/loader`, or any other option?